### PR TITLE
dsnd-600: Add additional Kafka headers and update insolvency mappings

### PR DESCRIPTION
**Pull request by kgeddis-ch**
Ticket: https://companieshouse.atlassian.net/browse/DSND-600

-Updated private-api-sdk-java version
-Updated Mapstruct mappings 
-Added updated_by field
-Updated unit tests

### CODE REVIEW

**Review by iichr — APPROVED** [review]
Nicely done especially with the weird date format we now need for delta_at

**Review by vchitti-kainos — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyMapper.java [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
-    @Mapping(target = "deltaAt", source = "deltaAt")
-    @Mapping(target = "cases", source = "caseNumbers")
+    @Mapping(target = "externalData.etag", ignore = true) // doesn't exist on source
+    @Mapping(target = "externalData.status", ignore = true) // doesn't exist on source
+    @Mapping(target = "internalData.deltaAt", ignore = true)
+    @Mapping(target = "externalData.cases", source = "caseNumbers")
     InternalCompanyInsolvency insolvencyDeltaToApi(Insolvency insolvency,
                                                    @Context String companyNumber);
+
+    /**
```
Comment (gseregni-ch) [inline src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyMapper.java:26]: I dowloaded the code and when I tried to mvn compile, it gave me a check style violation. 

Please see it below:
[WARNING] src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyMapper.java:[26] (javadoc) SummaryJavadoc: First sentence of Javadoc is incomplete (period is missing) or not present.

Can you please confirm if you have the same problem?

**Review by gseregni-ch — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyMapper.java (hunk 1 above)
Comment (dominicst-ch) [reply src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyMapper.java:26]: https://ci.platform.aws.chdev.org/teams/team-tiny-rebels/pipelines/insolvency-delta-consumer/jobs/analyse-pull-request/builds/29

Yep - fails in CI pipeline

Comment (kgeddis-ch) [reply src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyMapper.java:26]: Should be resolved now

**Review by gseregni-ch — APPROVED** [review]

**Pull request merged**